### PR TITLE
Add all missing reports + detailed column descriptions

### PR DESCRIPTION
**Pull request by aneesahahamed**

### CODE REVIEW

File: docs/records/reporting.md [hunk 1]
```diff
@@ -100,6 +116,22 @@ This report includes of the following columns:
...
+
+* **Created At**: The timestamp or date and time when a purchase order was created.
+* **PO number**: The source identifier or reference for the purchase order.
+* **Purchase Order ID**: The unique identifier for a purchase order.
+* **Vendor**: The name of the vendor or supplier associated with the purchase order.
+* **Total Price**: The total price of the purchase order.
+* **Total Units**: Total quantity of units ordered in the purchase order.
+* **Total Units Recieved**: Total quantity of units received against the purchase order.
+* **Status**: Current status of the purchase order.
+* **Source**: Source or origin of the purchase order.
```
Comment (emilieschario) [inline docs/records/reporting.md:133]:
```suggestion
* **Source**: Source or origin of the purchase order, can be Turbine or Anvyl
```

File: docs/records/reporting.md [hunk 2]
```diff
@@ -108,3 +140,67 @@ This report includes of the following columns:
...
 * **Remote SKU**: The remote ID associated with a SKU.
 * The projected demand for each week.
+
+## Revenue Recognition
+
+Details the process of recognizing revenue from sales transactions, aligning with US accounting standards to accurately reflect the company's financial performance over a specific period.
+
+This report includes of the following columns:
+
+* **Order Id**: Unique identifier assigned to each order within Turbine.
```
Comment (emilieschario) [inline docs/records/reporting.md:150]:
```suggestion
* **Order ID**: Unique identifier assigned to each order within Turbine.
```

File: docs/records/reporting.md [hunk 3]
```diff
@@ -79,6 +79,22 @@ This report includes of the following columns:
...
 * **Quantity**: The total number of units or items ordered in the specified transaction.
+
+## Sales Order Summary
+
+Overview of Sales Orders at the Orders level
+
+This report includes of the following columns:
+
+* **Created At**: The timestamp or date and time indicating when the order was created.
+* **Order ID**: A unique identifier assigned to each order for reference and tracking purposes.
```
[the remark this answers is not in the document]
Comment (aneesahahamed) [reply docs/records/reporting.md:90]: yes this is

File: docs/records/reporting.md [hunk 4]
```diff
@@ -79,6 +79,22 @@ This report includes of the following columns:
...
+* **Order Number**: The specific number associated with a particular order for easy identification.
+* **Channel**: The channel through which the order was received.
+* **Payment Status**: Indicates the current status of payment for the order, whether it is sent, completed, or any other relevant status.
+* **Fulfillment Status**: Indicates whether the order has been processed and fulfilled.
+* **Remote SKU**: The remote ID associated with a SKU.
```
[the remark this answers is not in the document]
Comment (aneesahahamed) [reply docs/records/reporting.md:95]: i'll make the changes

File: docs/records/reporting.md [hunk 5]
```diff
@@ -108,3 +140,67 @@ This report includes of the following columns:
...
+* **Revenue**: Total income generated from the corresponding order.
+* **Quantity**: The number of units included in the order.
+
+## Dismissed Exceptions
+
+Exceptions within the supply chain or production process that have been dismissed or resolved in Turbine
+
+This report includes of the following columns:
+
+* **ID**: Unique identifier assigned to each dismissed or resolved exception in Turbine's supply chain or production process.
```
[the remark this answers is not in the document]
Comment (aneesahahamed) [reply docs/records/reporting.md:167]: i'll make the change

**Review by emilieschario — APPROVED** [review]

**Pull request merged**